### DISCUSSION TITLE
Large library files

### DISCUSSION THREAD

**Discussion by trevorRO** [AxisCommunications/discussions#70] [ACAP] [answered]
I'm running in to an issue where I want to run an application on device using the artpec 7 chip, but my file size is too large for the onboard storage.  I've added a memory card to the camera, do you support either remapping the install location or allowing me to define a LD_LIBRARY_PATH to a location on the SD_CARD?  

Error

> 2022-09-29T10:34:26.119-06:00 axis-accc8ed92a4f [ NOTICE  ] file_upload[31687]: Failed to store file due to flash memory available
> 2022-09-29T10:34:35.742-06:00 axis-accc8ed92a4f [ CRIT    ] upload.cgi: Upload failed: ''
> ------------end of syslog, thanks for reading ---------------
> Could not upload package file, the file is too large or disk full

**Comment by linhataxis** [comment, marked as answer]
Hi,
We have recently released a new API in beta for installing applications on SD-card rather than the built-in flash partition. If you are interested in trying this API please send an email to acap@axis.com.
